module/userProjects/helper.js

```diff
@@ -1083,6 +1134,11 @@ module.exports = class UserProjectsHelper {
 			try {
 				let solutionExternalId = ''
 				let templateDocuments
+				let defaultACL = {
```

Comment:
@Prajwal17Tunerlabs you can deine default acl in common fileand use  it where ever you need

Reply:
defaultACL moved to common.js file